framework/components/ecl-language-list/ecl-language-list.twig

```diff
@@ -0,0 +1,92 @@
+{#
+  - modifier: the variant. Can be 'splash' or 'overlay' (default: 'splash')
+  - logo: display or not the European Commission logo. Can be true|false (default: false)
+#}
+
+{% set modifier = variant|default('splash') %}
```

Comment:
we'd better call it variant instead of modifier, as this is what we do in other components (I know, some components are still using "modifier"... but "variant" is the way to go)